corehq/apps/data_interfaces/interfaces.py

```diff
@@ -178,6 +178,7 @@ def selected(self):
         return FormManagementMode(self.request.GET.get(self.slug)).mode_name
 
 
+@location_safe
```

Comment:
:+1: 
Looks like the queries happen in `SubmitHistoryMixin`, which should be location safe already